packages/compass-preferences-model/src/preferences-schema.ts

```diff
@@ -5,6 +5,14 @@ import {
   featureFlags,
 } from './feature-flags';
 import { parseRecord } from './parse-record';
+import {
+  extractProxySecrets,
+  mergeProxySecrets,
+} from '@mongodb-js/devtools-proxy-support';
```

Comment:
```suggestion
} from '@mongodb-js/devtools-proxy-support/proxy-options';
```

added a commit to https://github.com/mongodb-js/devtools-shared/pull/451 to make these parts browser-compatible